code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -233,7 +233,7 @@
 		return
 	var/list/modules = list()
 	modules.Add(robot_module_types)
-	if(security_level == (SEC_LEVEL_RED || SEC_LEVEL_DELTA)) //There are no brakes in the combat module train.
+	if((crisis_override && security_level == SEC_LEVEL_RED) || SEC_LEVEL_DELTA) //no fun allowed anymore.

```

Comment:
`if ((condition AND condition) OR TRUE)` someone correct me if I'm wrong but this statement will always pass without fail.

I think it needs to be `if((crisis_override && security_level == SEC_LEVEL_RED) || security_level == SEC_LEVEL_DELTA)`